src/components/CtaSection/Cta.scss

```diff
@@ -0,0 +1,32 @@
+@use '../../sass/abstracts/variables';
+
+.cta {
+  margin-top: 80px;
+}
+
+.cta_image {
+  width: 100%;
+  margin-bottom: 40px;
+}
+
+.cta_details {
+  align-items: center;
+  text-align: center;
+}
+
+h1 {
```

Comment:
This is a bit dangerous. You may override some previously set h1 styling here as it will be compiled in a single css file...

Reply:
@gmagnenat , So i should have a different element instead of an H1 ?

Reply:
it's a good practice to have only one H1 on your page anyway and don't use html tags for the look they have.
"I need a big title, I put a h1" but use semantic elements and style them how you want them to look.
As CSS cascade (top to bottom) the issue was that you were changing color for all <p> tag so it overrides what you previously added. If you scope it more precisely with a class or by targetting <p> inside cta you avoid that problem.